lib/config/config.js

```diff
@@ -203,6 +199,13 @@ class Config {
 
         this.language = plugins[languagePluginName].languages[localLanguageName];
 
+        if (this.language.defaultLanguageOptions || languageOptions) {
```

Comment:
We only need to check if it's undefined so I think `??` is more appropriate.

```suggestion
        if (this.language.defaultLanguageOptions ?? languageOptions) {
```